src/sage/categories/category.py

```diff
@@ -928,8 +932,8 @@ def _set_of_super_categories(self):
 
         TESTS::
 
-            sage: C = HopfAlgebrasWithBasis(GF(7))
-            sage: C._set_of_super_categories == frozenset(C._all_super_categories_proper)
+            sage: C = HopfAlgebrasWithBasis(GF(7))                                                                      # optional - sage.libs.pari
+            sage: C._set_of_super_categories == frozenset(C._all_super_categories_proper)                               # optional - sage.libs.pari
```

Comment:
```suggestion
            sage: C = HopfAlgebrasWithBasis(GF(7))                                     # optional - sage.libs.pari
            sage: C._set_of_super_categories == set(C._all_super_categories_proper)    # optional - sage.libs.pari
```

Reply:
There is no difference comparing sets and frozensets, and this gives you the space you need to reach column 88.